tool/typedb3/install-server.sh

```diff
@@ -5,7 +5,8 @@ source tool/typedb3/config.sh
 DISTRIBUTION="typedb-all-linux-x86_64"
 DISTRIBUTION_DIR="$DISTRIBUTION-$SERVER_VERSION"
 DISTRIBUTION_TARGZ="$DISTRIBUTION_DIR.tar.gz"
-DISTRIBUTION_URL="https://repo.typedb.com/public/public-snapshot/raw/names/$DISTRIBUTION/versions/$SERVER_VERSION/$DISTRIBUTION_TARGZ"
+DISTRIBUTION_URL_SNAPSHOT="https://repo.typedb.com/public/public-snapshot/raw/names/$DISTRIBUTION/versions/$SERVER_VERSION/$DISTRIBUTION_TARGZ"
+DISTRIBUTION_URL_RELEASE="https://repo.typedb.com/public/public-release/raw/names/$DISTRIBUTION/versions/$SERVER_VERSION/$DISTRIBUTION_TARGZ"
 
-curl -o "$DISTRIBUTION_TARGZ" "$DISTRIBUTION_URL"
-tar -xf "$DISTRIBUTION_TARGZ"
+(curl -o "$DISTRIBUTION_TARGZ" "$DISTRIBUTION_URL_SNAPSHOT" && tar -xf "$DISTRIBUTION_TARGZ") ||
+    (curl -o "$DISTRIBUTION_TARGZ" "$DISTRIBUTION_URL_RELEASE" && tar -xf "$DISTRIBUTION_TARGZ")
```

Comment:
Try downloading the version given from the snapshot url, retry from the release url on failure.